scss/comments.scss

```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
         margin-top: 10px;
     }
 }
+
+// A published comment
+.comment {
+    clear:both;
+    list-style-type: none;
+    overflow: hidden;
+    padding: 20px 0 10px;
+    position: relative;
+
+    &.author-comment {
+        &:after {
+          content: 'Author';
+          float: right;
+          position: absolute;
+          top: 1.5em;
+          right: 0.5em;
+          font-size: 1em;
+          font-weight: bold;
+      }
+
+      > .comment-text {
+        border: 1px solid blue;
+    }
+
+}   
+
+&.spam {
+    .comment-text {
+        border: 1px dashed orange;
+    }
+}
+
+.comment-text {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    box-shadow: none;
+    margin: 0;
+    padding: 0 15% 0 20px;
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: pre-line;
+    word-wrap: break-word;
+
+    p:last-child {
+        margin-bottom: 0;
+    }
+}
+
+.date {

```

Comment:
I think this is too generic a style to have at the top level. There could be any `date` in the html not related to comments. Can we nest these similar to the previous styles, in `.comments-holder-container`?